client/src/shared/components/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Link from 'next/link';
+
+const SideBar: React.FunctionComponent = () => {
+  return (
+    <div className='space-y-9'>
```

Comment:
as for standard, use double quote for html properties 
```suggestion
    <div className="space-y-9'">
```